eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -65,8 +65,7 @@ case class NodeParams(keyManager: KeyManager,
                       pingInterval: FiniteDuration,
                       pingTimeout: FiniteDuration,
                       pingDisconnect: Boolean,
-                      maxFeerateMismatch: Double,
-                      updateFeeMinDiffRatio: Double,
+                      onChainFeeConf: OnChainFeeConf,
```

Comment:
Last nit: `onChainFeeConf` is lost in between peer-specific params, can you move it somewhere where it makes more sense?

Reply:
Done in 36a8d5e